app/controllers/schemes_controller.rb

```diff
@@ -25,9 +25,16 @@ def new
   end
 
   def create
-    @scheme = Scheme.new(scheme_params)
-    if @scheme.save
-      render "schemes/primary_client_group"
+    @scheme = Scheme.new(scheme_params.except(:support_services_provider_before_type_cast))
```

Comment:
If you move the `.except` down into the params method you can dry this up a bit still?